src/main/java/games/strategy/engine/framework/GameDataFileUtils.java

```diff
@@ -9,21 +9,22 @@
 
 import com.google.common.annotations.VisibleForTesting;
 
-import games.strategy.engine.ClientContext;
+import games.strategy.triplea.settings.ClientSetting;
 
 /**
  * A collection of utilities for working with game data files.
  */
 public final class GameDataFileUtils {
   private static final IOCase DEFAULT_IO_CASE = IOCase.SYSTEM;
 
-  private static final SaveGameFormat DEFAULT_SAVE_GAME_FORMAT =
```

Comment:
This class field was changed to a method because the feature toggle can now change dynamically at runtime.